src/components/Spinner/index.vue

```diff
@@ -0,0 +1,29 @@
+<template src="./index.pug" lang="pug"></template>
```

Comment:
Don't import the pug, write it in the `.vue`

Reply:
Why?